rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1091,9 +1094,15 @@ def test_upgrade_db_19_to_20(user_data_dir):
     query = cursor.execute('SELECT xpub, derivation_path, label FROM xpubs;')
     length = 0
     expected_results = [
-        ('xpub6DCi5iJ57ZPd5qPzvTm5hUt6X23TJdh9H4NjNsNbt7t7UuTMJfawQWsdWRFhfLwkiMkB1rQ4ZJWLB9YBnzR7kbs9N8b2PsKZgKUHQm1X4or', '', 'My totally public xpub'),  # noqa: E501
-        ('ypub6WkRUvNhspMCJLiLgeP7oL1pzrJ6wA2tpwsKtXnbmpdAGmHHcC6FeZeF4VurGU14dSjGpF2xLavPhgvCQeXd6JxYgSfbaD1wSUi2XmEsx33', '', ''),  # noqa: E501
-        ('xpub68V4ZQQ62mea7ZUKn2urQu47Bdn2Wr7SxrBxBDDwE3kjytj361YBGSKDT4WoBrE5htrSB8eAMe59NPnKrcAbiv2veN5GQUmfdjRddD1Hxrk', 'm/1/15', 'valid derivation path xpub'),  # noqa: E501
+        (
+        'xpub6DCi5iJ57ZPd5qPzvTm5hUt6X23TJdh9H4NjNsNbt7t7UuTMJfawQWsdWRFhfLwkiMkB1rQ4ZJWLB9YBnzR7kbs9N8b2PsKZgKUHQm1X4or',
+        '', 'My totally public xpub'),  # noqa: E501
+        (
+        'ypub6WkRUvNhspMCJLiLgeP7oL1pzrJ6wA2tpwsKtXnbmpdAGmHHcC6FeZeF4VurGU14dSjGpF2xLavPhgvCQeXd6JxYgSfbaD1wSUi2XmEsx33',
+        '', ''),  # noqa: E501
+        (
+        'xpub68V4ZQQ62mea7ZUKn2urQu47Bdn2Wr7SxrBxBDDwE3kjytj361YBGSKDT4WoBrE5htrSB8eAMe59NPnKrcAbiv2veN5GQUmfdjRddD1Hxrk',
+        'm/1/15', 'valid derivation path xpub'),  # noqa: E501
```

Comment:
I believe this has been formatted by your editor. We keep one line per item in the tuple and also indented but is fine to have them in one line as before. Also if you make this change you need to remove the `# noqa: E501` (ignore long line)